src/howitz/endpoints.py

```diff
@@ -63,6 +63,8 @@ def auth_handler(username, password):
             session["not_connected_counter"] = 0
             return user
 
+        raise Exception('Unexpected error on Zino authentication')
+
```

Comment:
Nah, raise an Not Authenticated again, or a Zino-specific error that is handled like a 403.

`Exception` exists to be inherited from, not to be raised or caught directly =)